ios-baseUITests/Info.plist

```diff
@@ -17,6 +17,6 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>1.0</string>
```

Comment:
This should be just a number, we have `CFBundleShortVersionString` for semantic versioning.